code/game/objects/items/devices/powersink.dm

```diff
@@ -82,15 +82,15 @@
 				attached = locate() in T
 				if(!attached)
 					to_chat(user, span_warning("\The [src] must be placed over an exposed, powered cable node!"))
-				else
+				else if(do_after(user, 5 SECONDS, target = src))
```

Comment:
```suggestion
				else if(do_after(user, 5 SECONDS, target = src) && attached)
```
Also `attached` has to be set back to null if it can't be wrenched down.